balena/auth.py

```diff
@@ -12,6 +12,7 @@ class Auth(object):
     """
 
     _user_detail_cache = {}
+    _user_full_detail_cache = {}
```

Comment:
I guess we should also clean this up from within `login`, `login_with_token` & `log_out`, like we do for `_user_detail_cache` , right?

Reply:
updated.